src/__tests__/test.ts

```diff
@@ -61,6 +82,26 @@ describe('WellPlate', () => {
     ).toStrictEqual(expected);
   });
 
+  it('getCodeRange by rows, iteration order by rows', () => {
+    const wellPlate = getWellPlate(
+      '4x6',
+      PositionFormat.LetterNumber,
+      IterationOrder.ByRow,
+    );
+    const expected = ['C5', 'C6', 'D1', 'D2', 'D3'];
```

Comment:
@stropitek My apologies for the late review comment, but better late than never. You've set `IterationOrder.ByRow`  here but this result of `getCodeRange` seems to be going _across_ (AKA `ByColumn`), no?

For example, for `wellPlate.getPositionCodeRange(11, 18)`, I would expect the result to be:

```
['B6', 'B7'....'C6', 'D1']
```

(assuming `getPositionCodeRange` is meant to be inclusive of both indices, I couldn't actually see in docs if that's supposed to be the case).

![image](https://user-images.githubusercontent.com/1213121/111949233-73a9cc00-8b34-11eb-9d09-95065f9351c9.png)

Reply:
Sorry, my above expected value is not correct, I believe it should instead be:

```
['D3', 'A4', 'B4', 'C4', 'D4', 'A5', 'B5', 'C5']
```

Reply:
I see quite a lot has changed since this PR was merged @stropitek , please ignore my comments 👍 .

Reply:
I think you have good point though. My rationale was to name things similarly to the `IndexationOrder`. option. So "by row" selects a range by iterating on the rows first, but that effectively selects columns! Makes sense but kind of convoluted and misleading.

I'm gonna rename that to `SubsetMode.Columns` and `SubsetMode.Rows`

Reply:
Yeah naming these well plate iteration orders in an intuitive way that avoids misintepretation is a real challenge. I find myself constantly spelling it out (e.g. `...we go down then across columns (i.e. A1, B1....H1, A2, B2 etc)`) when I'm writing code or documentation related to iteration orders.